plugin/trino-elasticsearch/src/test/java/io/trino/plugin/elasticsearch/BaseElasticsearchConnectorTest.java

```diff
@@ -459,6 +461,292 @@ public void testArrayFields()
                 "VALUES ('hello', 20, 30, 60, NULL)");
     }
 
+    @Test
```

Comment:
Just a NOTE, We should split tests for each type.  This can be handled in another PR altogether.

Reply:
Let's open a separate issue with this to refactor all the tests in Elasticsearch. We should make the Elasticsearch test suite a bit more modular over all.